api/GetRainfallByAddress/__init__.py

```diff
@@ -7,6 +7,26 @@
 from roof_segmentation import roof_segmentation
 from consumption_helper import get_water_usage
 
+def calc_water_savings(roof_size, avg_rainfall):
+    return roof_size * avg_rainfall # sqm * mm = L
+
+def potential_savings(monthly_consumption, annual_rain, roof_surface_area_in_square_meters):
+
+    # Get how much water we can save
+    water_harvest_potential = calc_water_savings(roof_surface_area_in_square_meters, annual_rain) # in litres
+
+    # Calculate average daily consumption
+    avg_usage = []
+    for month in monthly_consumption:
+        avg_usage.append(month["TotalConsumed"]/30)
+    avg_usage = sum(avg_usage)/len(avg_usage)
+
+    # calculate potiential savings
+    # NOTE: 822L threshold is based on UnityWater's pricing tiers
+    potential_savings =  water_harvest_potential * 0.667 if (avg_usage > 822) else water_harvest_potential * 1.333
```

Comment:
Will literage under 822L be charged at the lower or higher rate? This will charge all litres as the higher price if it exceeds 822 which I believe wouldn't be correct (I'd need to check pricing)

Reply:
Great pick up, it should have been the other way around